OWNERS_ALIASES

```diff
@@ -1,5 +1,7 @@
 aliases:
   sig-api-machinery-leads:
+    - deads2k
```

Comment:
Why twice?

Reply:
Seems like a bug in `make generate` (I didn't manually edit this file).

Reply:
Names new to be alphabetically ordered. I assume the bug is around that.

Reply:
I thought the bug is because it just generates this list from sigs.yaml but doesn't dedup in cases where someone is a TL and a chair

Reply:
https://github.com/kubernetes/community/blob/master/generator/aliases.tmpl#L4-L9

Reply:
Not that it is necessarily a bug

Reply:
He's listed twice (once in each section) so he gets put in here twice. It's just a dumb go text template.